src/javascript/binary/websocket_pages/user/ico_portfolio.js

```diff
@@ -107,6 +107,35 @@ const ICOPortfolio = (() => {
         is_first_response = false;
     };
 
+    const cancelBid = (contract_id) => {
+        const lightbox = $('#cancel_bid_confirmation');
+        document.body.appendChild(lightbox[0]);
+        lightbox.setVisibility(1);
+        lightbox.find('.error-msg').addClass('invisible');
+        lightbox.find('#chk_confirm').prop('checked', false);
+
+        const confirm_form_id = '#frm_confirm';
+        if (!lightbox.find(confirm_form_id).hasClass('validation_initialized')) {
+            Validation.init(confirm_form_id, [
+                { selector: '#chk_confirm', validations: [['req', { hide_asterisk: true }]] },
```

Comment:
do we need the nested array for validations?

Reply:
@kellybinary Just checked the `form_validation.js`, Since we are passing an option(`{hide_asterisk: true}`) it should be a nester array!